improver/wind_calculations/wind_direction.py

```diff
@@ -123,14 +126,7 @@ def deg_to_complex(
             3D array or float - wind direction translated to
             complex numbers.
         """
-        # Convert from degrees to radians.
-        angle_rad = np.deg2rad(angle_deg)
-        # Derive real and imaginary components (also known as a and b)
-        real = radius * np.cos(angle_rad)
-        imag = radius * np.sin(angle_rad)
-
-        # Combine components into a complex number and return.
-        return real + 1j * imag
+        return deg_to_complex(angle_deg, radius)
```

Comment:
defining these elsewhere to avoid circular imports

Reply:
Well, there are numerous unittests relating to the wind stuff involving the use of these things which are tied to the wind data.  I can remove if you are happy that the tests I have written for these functions are enough??